redisson/src/main/java/org/redisson/RedissonList.java

```diff
@@ -24,18 +24,9 @@
 import static org.redisson.client.protocol.RedisCommands.LREM_SINGLE;
 import static org.redisson.client.protocol.RedisCommands.RPUSH_BOOLEAN;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.NoSuchElementException;
-
-import org.redisson.api.RFuture;
-import org.redisson.api.RList;
-import org.redisson.api.RedissonClient;
-import org.redisson.api.SortOrder;
+import java.util.*;
+
+import org.redisson.api.*;
```

Comment:
It looks that your IDE settings overridde previous code style. It is common practice to prevent star imports.